pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/loaders/ReportDeadLetterLoader.kt

```diff
@@ -0,0 +1,103 @@
+package gov.cdc.ocio.processingstatusapi.loaders
+
+import com.azure.cosmos.models.CosmosQueryRequestOptions
+import gov.cdc.ocio.processingstatusapi.models.ReportDeadLetter
+import gov.cdc.ocio.processingstatusapi.models.dao.ReportDao
+import java.text.SimpleDateFormat
+import java.time.ZoneOffset
+import java.util.*
+
+class ReportDeadLetterLoader : CosmosDeadLetterLoader() {
+
+    fun getByUploadId(uploadId: String): List<ReportDeadLetter> {
```

Comment:
Please add function and class headers to all files.

Reply:
Done. Please review